README.md

```diff
@@ -63,6 +71,13 @@ int training_steps = 1000;
 float learning_rate = 0.01f;
 int display_step = 100;
 
+// Sample data
+train_X = np.array(3.3f, 4.4f, 5.5f, 6.71f, 6.93f, 4.168f, 9.779f, 6.182f, 7.59f, 2.167f,
```

Comment:
Shouldn't be `var train_X = np.array()`?

Reply:
Not sure, I took it from here:     https://github.com/SciSharp/SciSharp-Stack-Examples/blob/bc4eea022a3d047f0e96e994713ea398b286947c/src/TensorFlowNET.Examples/BasicModels/LinearRegressionEager.cs#L98

Reply:
The C# notebook example here also does not have `var`:
https://github.com/SciSharp/SciSharpCube/blob/master/home/samples/LinearRegression.ipynb